pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.isort]
+known_third_party = ["PIL", "attr", "cv2", "gym", "habitat_sim", "imageio", "matplotlib", "numba", "numpy", "orbslam2", "pytest", "quaternion", "requests", "scipy", "setuptools", "torch", "torchvision", "tqdm", "version", "yacs"]
+multi_line_output = 3
+force_grid_wrap = false
+line_length = 79
+include_trailing_comma = true
+use_parentheses = true
+
+[tool.black]
+line_length = 79
+exclude = '''
```

Comment:
Is there a possibility of picking this up from `.gitignore`?

Reply:
black doesn't support [that feature yet](https://github.com/mdagaro/isort/commit/af012ef18e4680f2b3780d36bca9e3f25fa19a60), but it should run on files that are part of repo on `pre-commit` hook.